includes/REST_API/Hotlinking_Controller.php

```diff
@@ -429,7 +429,7 @@ public function get_item_schema(): array {
 			return $this->add_additional_fields_schema( $this->schema );
 		}
 
-		$allowed_mime_types = $this->types->get_allowed_mime_types();
+		$allowed_mime_types = $this->get_media_types();
```

Comment:
Wondering about the naming here -- any reason not to say get_mime_types here?

Code says it returns an `Array of supported media types.` and it does .... but the variable says $mime_type --- so feels like we're just switching back and forth between mime types or media types .. bit of metal overhead we could avoid?

Reply:
Rename to get_allowed_mime_types to be clear.